lib/sqlalchemy/testing/suite/test_reflection.py

```diff
@@ -1148,6 +1148,72 @@ def test_autoincrement_col(self):
             assert id_.get("autoincrement", True)
 
 
+class ComputedReflectionTest(fixtures.TablesTest):
+    run_inserts = run_deletes = None
+
+    __backend__ = True
+
+    @classmethod
+    def define_tables(cls, metadata):
+        if testing.requires.computed_columns.enabled:
+            Table(
+                "computed_column_table",
+                metadata,
+                Column("id", Integer, primary_key=True),
+                Column("normal", Integer),
+                Column("computed_col", Integer, sa.Computed("normal + 42")),
+                Column("with_default", Integer, server_default="42"),
+            )
+
+    @testing.requires.computed_columns
+    @testing.requires.table_reflection
+    def test_computed_col_default_not_set(self):
+        insp = inspect(testing.db)
+
+        cols = insp.get_columns("computed_column_table")
+        for col in cols:
+            if col["name"] == "with_default":
+                is_true("42" in col["default"])
+            elif not col["autoincrement"]:
+                is_(col["default"], None)
+
+    @testing.requires.computed_columns
+    @testing.requires.table_reflection
+    def test_get_column_returns_computed(self):
```

Comment:
this if filled with `if`, `esif` for each dialect. Maybe we should just move the dialect dependent part of this test in the test_reflection of each dialect